### DISCUSSION TITLE
Custom remote file for DirecTV H24

### DISCUSSION THREAD

**Discussion by tmcafee52** [GerryDazoo/Slinger#115] [General]
I modified the remote.txt file to work with the DirecTV H24 IR remote.  That remote works with my H21, H24 and H25 receivers and my HR24 and Genie HR54 DVR's.  I've attached the file: remote_directv_h24.txt.  I was unable to figure out how to handle the TV Input button which changes sources.  Also, the Restart button kills the slingbox_server.exe process but doesn't restart it.   

[remote_directv-h24.txt](https://github.com/GerryDazoo/Slinger/files/9721783/remote_directv-h24.txt)

**Comment by sp222222** [comment]
thank ! I will compare this to the one I'm using  on my Genie.
I never have used the TV input button on that remote.

**Comment by GerryDazoo** [comment]
You need to be running the slingbox server as a service to get it to restart. See the readme. There's a simple batch file for windows and some instructions for Unix. Feel free to rename the key if you find the name misleading.

**Comment by tmcafee52** [comment]
Thanks Gerry.   I added the shortcut to the RunSling.bat batch file to the startup directory and the Restart function works fine now.  I made a couple of minor formatting changes to the remote_directv_h24.txt file also.  

[remote_directv-h24.txt](https://github.com/GerryDazoo/Slinger/files/9728986/remote_directv-h24.txt)

**Comment by tmcafee52** [comment]
I thought that I'd let everyone know about a problem that I had trying to get my ProHD to work with the remote control.  I got my 350 to work first but couldn't get the remote to work on the ProHD.  I had to change the line in the Remote section of the config.ini file to "Code=3".

**Reply by GerryDazoo** [reply]
This is not ProHD specific. The "code" selects the slingbox remote to used for the configured VideoSource. All video sources can have their own remote definitions. In your case the VideoSource that you needed was "3" and not one of the other inputs,

**Comment by GerryDazoo** [comment]
Yes, I recommend setting the VideoSource and code to the same value. If you don’t set the video source it will use whatever the last source was and that might not be the right one of the remote code supplied.

**Comment by tmcafee52** [comment]
I created a modified DirecTV H24 remote file with larger rectangular digit buttons and minor color changes.

[remote_directv-h24_large_digits.txt](https://github.com/GerryDazoo/Slinger/files/9905397/remote_directv-h24_large_digits.txt)

**Comment by tmcafee52** [comment]
Minor corrections to the Pg+, Pg-, Ch+ and Ch- buttons for the DirecTV H24 remote file. 
[remote_directv-h24_large_digits.txt](https://github.com/GerryDazoo/Slinger/files/10242220/remote_directv-h24_large_digits.txt)
